.github/workflows/velox_be.yml

```diff
@@ -204,7 +204,7 @@ jobs:
       - name: Build and Run unit test for Spark 3.3.1
         run: |
           docker exec velox-ubuntu2004-test-spark33-$GITHUB_RUN_ID bash -l -c 'cd /opt/gluten && \
-          mvn clean install -Pspark-3.3 -Pbackends-velox -Prss -Pspark-ut  -DargLine="-Dspark.test.home=/var/cache/spark33/spark331"'
+          mvn clean install -Pspark-3.3 -Pbackends-velox -Prss -Pspark-ut -DargLine="-Dspark.test.home=/var/cache/spark33/spark331" -DtagsToInclude=org.apache.spark.tags.ExtendedSQLTest'
```

Comment:
`-DtagsToInclude=org.apache.spark.tags.ExtendedSQLTest` will only pick the suites with this tag, should be removed?